lib/features/neutral-element-colors/NeutralElementColors.js

```diff
@@ -0,0 +1,35 @@
+import {
+  TOGGLE_MODE_EVENT
+} from '../../util/EventHelper';
+
+const ID = 'neutral-element-colors';
+
+export default function NeutralElementColors(
+    eventBus, elementRegistry, elementColors) {
+
+  this._elementRegistry = elementRegistry;
+  this._elementColors = elementColors;
+
+  eventBus.on(TOGGLE_MODE_EVENT, event => {
+    const { active } = event;
+
+    if (active) {
+      this._setNeutralColors();
+    }
+  });
+}
+
+NeutralElementColors.prototype._setNeutralColors = function() {
+  this._elementRegistry.forEach(element => {
+    this._elementColors.add(element, ID, {
```

Comment:
Where do we get these magic numbers from?

We export utilities to retrieve these colors via `BpmnRenderUtil#get*`

Reply:
Makes sense, I'll change that.

Reply:
Why would we want to set neutral colors in the first place? Maybe we can address this in another way (or stick to the magicies!).

Reply:
Well, we don't have to, but the original idea (back in 2017) was that removing the custom color makes it possible to use color to show things like which sequence flow is selected or which scope is selected. Otherwise, we'd have to use something other than just color to do that.

Reply:
I think I understand now. 

The purpose is to reset (clearing custom colors) once token simulation activates?

Reply:
Exactly.